.changeset/fair-tigers-eat.md

```diff
@@ -1,8 +1,6 @@
 ---
-"@lux-design-system/components-react": major
+"@lux-design-system/components-react": minor
```

Comment:
Helaas is dat niet waar, we gaan van 1.0.0-alpha naar 1.0.0, dit is volgens changesets je eerste major upgrade.